### PR TITLE
free field

### PR DESCRIPTION
**Pull request by tzipisheinin**

### CODE REVIEW

**Comment by tzipisheinin** [comment]
[(3) Run test with freefield defintion](https://trello.com/c/GSbQjP6u/3723-3-run-test-with-freefield-defintion)

**Comment by tzipisheinin** [comment]
[(3) Run test with freefield defintion](https://trello.com/c/GSbQjP6u/3723-3-run-test-with-freefield-defintion)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@audyx/audyx-deps",
-  "version": "1.0.69",
+  "version": "1.0.70-free-field-test",
```
Comment (mhashel) [inline package.json]: remember to change...

File: src/cljs/audyx/test/transducers.cljs [hunk 2]
```diff
@@ -105,7 +103,7 @@
 
 (defmethod test-speakers :default [{:keys [transducer hearing-context routing]}]
   (case transducer
-    "FreeField" (audio-context/available-speakers :general (get-in routing [:default :voice]))
+    "FreeField" (get-in routing [:default :voice])
```
Comment (mhashel) [inline src/cljs/audyx/test/transducers.cljs]: revert

File: src/cljs/audyx/audio_engine/tonal_constraints.cljs [hunk 3]
```diff
@@ -13,9 +13,11 @@
 
 (defn _max-dbhl [signal-type tonal-calib transducer-definition hz]
   (if-let [max-dbhl (_max-transducer-dbhl-for-frequency signal-type transducer-definition hz)]
+    (do
```
Comment (mhashel) [inline src/cljs/audyx/audio_engine/tonal_constraints.cljs]: remove this file from commit

**Pull request merged**